mandatory/3-function-output.js

```diff
@@ -1,16 +1,17 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
   return Math.random() * 10;
-}
+} // This function generates a random number between 0 and 1 (including the number 0 but not the number 1). It then multiplies that number by 10.
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
-}
+} // This function combines two different words into one. The end result being that "word1" and "word2" now become "word1word2".
 
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  return firstWord.concat(secondWord) + secondWord.concat(thirdWord);
```

Comment:
So what went wrong here is when you evaluate the two expressions, you're actually returning this
```javascript
return "firstWordsecondWord" + "secondWordthirdWord"
```
If that makes sense?

You want something more akin to 
```javascript
return firstWord.concat(secondWord).concat(thirdWord)
```
And additionally some spaces.

Reply:
Concat also lets you do things like

```javascript
someWord.concat(" ", otherWord, " ", "and", "other stuff")
```